Source/Chromosomes/Vector/randomization_mutation.cpp

```diff
@@ -1,14 +1,14 @@
 #include "vector_chromosome.h"
 #include "randomization_mutation.h"
 
-randomization_mutation::randomization_mutation(const randomization_mutation_metadata *metadata, stochastic *stochastic_instance) :
+randomization_mutation::randomization_mutation(const randomization_mutation_metadata* const metadata, stochastic *stochastic_instance) :
     mutation_internal(metadata){
     _stochastic_instance = stochastic_instance;
 }
 
 randomization_mutation::~randomization_mutation() = default;
 
-chromosome *randomization_mutation::invoke(vector_chromosome *chromosome) {
+chromosome *randomization_mutation::invoke(const vector_chromosome* const chromosome) const {
     auto* child = new vector_chromosome(chromosome);
```

Comment:
should `auto` be marked with a `const`

Reply:
this can only be `auto* const` since we call set_genes it can't be `const auto* const`